components/location/default/index.mjs

```diff
@@ -1,34 +1,54 @@
 import { InternalAppmapError } from "../../error/index.mjs";
```

Comment:
I don't understand the need for supporting undefined line and/or column. Plus it seems to me that your format does not differentiate between `{line: 123, column: undefined}` and `{line: undefined, column: 123}`.

Reply:
This is because the appmap format calls for just the line in the location, but I suppose I can avoid modifying the location code by keeping the column and snipping it off at the very end.